spec/simpleJSONataFilter.spec.js

```diff
@@ -9,7 +9,7 @@ const chaiAsPromised = require('chai-as-promised');
 chai.use(chaiAsPromised);
 
 describe('Test filter', () => {
-    const msg = {
+    let msg = {
```

Comment:
You changed this line from `const` to `let` but I don't see you modifying the value.